### ISSUE TITLE
UI Structure: QuestionFragment [Blocked: #112, #120, #150]

### ISSUE THREAD

**Issue by BenHenning** [oppia/oppia-android#158]
This is tracking implementing the low-fidelity version of the QuestionFragment whose full implementation is tracked in #27.

**Comment by BenHenning** [comment]
This is blocked on #112, #118, #120, and #150.

**Comment by BenHenning** [comment]
Open question: do we also need a question activity?

**Comment by BenHenning** [comment]
This is effectively a duplicate of #159.
